stage0/layout.ld

```diff
@@ -15,7 +15,7 @@
 */
 
 HIDDEN(TOP = 4096M);
-HIDDEN(BIOS_SIZE = 256K);
+HIDDEN(BIOS_SIZE = 512K);
```

Comment:
I suspect this would once-again require updating the readme to keep in sync.

Reply:
I've just got rid of the explicit size in the README :)